lib/ApplicationFeatures/LanguageFeature.cpp

```diff
@@ -126,9 +126,11 @@ void LanguageFeature::collectOptions(
       new StringParameter(&_defaultLanguage),
       arangodb::options::makeDefaultFlags(arangodb::options::Flags::Hidden));
```

Comment:
Should we add here info about deprecation of this option in 3.10?

Reply:
Not in 3.9 branch.